redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -211,7 +211,7 @@ def _check_db(self):
                 config = conn.config_get("maxclients")
             except redis.ResponseError:
                 # config_get is disabled on some environments
-                self.log.debug("Error querying config")
+                self.log.warning("Unable to collect max clients: CONFIG GET disabled in managed Redis instances.")
```

Comment:
Let's keep it as `debug` since there is no way to hide the log line for people using a cloud version of Redis